src/Sylius/Bundle/ShopBundle/Resources/config/services/email.xml

```diff
@@ -19,14 +19,12 @@
     <services>
         <defaults public="true" />
 
-        <service id="sylius.mailer.contact_email_manager.shop" class="Sylius\Bundle\CoreBundle\Mailer\ContactEmailManager">
+        <service id="sylius_shop.mailer.email_manager.contact" class="Sylius\Bundle\CoreBundle\Mailer\ContactEmailManager">
```

Comment:
I've just noticed, that it is not consistent with names in AdminBundle: https://github.com/Sylius/Sylius/pull/17199/files#diff-51e5b98722170003f7d20928f5caa6ec5c2a23fb4eaee804bbf174a9da7e994dR38 👀, make it consistent, but probably in a separate PR 🖖🏻